ui/frontend/editor/Editor.tsx

```diff
@@ -118,7 +119,7 @@ const Editor: React.SFC = () => {
   const execute = useCallback(() => dispatch(actions.performPrimaryAction()), [dispatch]);
   const onEditCode = useCallback((c) => dispatch(actions.editCode(c)), [dispatch]);
 
-  const SelectedEditor = editor === EditorType.Simple ? SimpleEditor : AdvancedEditor;
+  const SelectedEditor = editor === EditorType.Simple ? SimpleEditor : editor === EditorType.Monaco ? MonacoEditor : AceEditor;
```

Comment:
My preference once we get > 2 items is to make a little lookup object with the enum as keys.